backend/protondrive/protondrive.go

```diff
@@ -62,13 +62,17 @@ func init() {
 		NewFs:       NewFs,
 		Options: []fs.Option{{
 			Name:     "username",
-			Help:     `The username of your proton drive account`,
+			Help:     `The username of your proton account`,
 			Required: true,
 		}, {
 			Name:       "password",
-			Help:       "The password of your proton drive account.",
+			Help:       "The password of your proton account.",
 			Required:   true,
 			IsPassword: true,
+		}, {
+			Name:       "mailboxPassword",
+			Help:       "The mailbox password of your two-password proton account.",
+			IsPassword: true,
```

Comment:
After second thought, I will move it to Advance, since Proton's main user base is using 1-password mode!

I will add the reference KB URL in the text!